DETAILED_INSTRUCTIONS.md

```diff
@@ -104,6 +106,13 @@ Creating Xpath file: junos-qfx-conf-protocols@2019-01-01_xpath.txt
 Creating Xpath file: junos-qfx-conf-protocols@2019-01-01_xpath.xml
 ```
 
+Note: Copy the below yang files from the common directory as the other junos yang modules such as junos, junos-es, junos-nfx etc are dependent on them
```

Comment:
Again, please uppercase YANG. It's worth commenting that the dependent YANG files must be from the same set of files. You can't mix and match dependent files from another version, despite the naming being the same.